modules/creating-a-drift-metric-using-cli.adoc

```diff
@@ -0,0 +1,63 @@
+:_module-type: PROCEDURE
+
+[id='creating-a-drift-metric-using-cli_{context}']
+= Creating a data drift metric by using the CLI
+
+[role='_abstract']
+You can use the OpenShift command-line interface (CLI) to create a data drift metric for a model.
+
+.Prerequisites
+
+* You are familiar with the specific data set schema and understand the names and meanings of the inputs and outputs.
```

Comment:
The following could be shorted:

"...understand the names and meanings of the inputs and outputs."

"...and understand the relevant inputs and outputs."